src/sentry/static/sentry/app/views/settings/incidentRules/triggers/actionsPanel/index.tsx

```diff
@@ -262,13 +268,83 @@ class ActionsPanel extends React.PureComponent<Props> {
             return (
               actions &&
               actions.map((action: Action, i: number) => {
-                const isUser = action.targetType === TargetType.USER;
-                const isTeam = action.targetType === TargetType.TEAM;
-
                 const availableAction = availableActions?.find(
-                  a => this.getActionUniqueKey(a) === this.getActionUniqueKey(action)
+                  a => getActionUniqueKey(a) === getActionUniqueKey(action)
                 );
 
+                let field0 = <span />;
+                let field1 = <span />;
```

Comment:
@mgaeta I find this hard to read. Wondering if we could just have two pure functions that return `field0` and `field` respectively